### ISSUE TITLE
Should the stopwatch pause and reset when unchecked?

### ISSUE THREAD

**Issue by samreid** [phetsims/pendulum-lab#213]
While I was reviewing the CCK AC design doc, it referred me to look at the stopwatch in pendulum lab.  I noticed that checking "Stopwatch", then starting the stopwatch, then unchecking "Stopwatch" neither pauses nor restarts it.  However, the Wave Interference design called for the stopwatch to be paused and reset when dropping back into its toolbox.  Should the pendulum lab stopwatch pause and reset when unchecked?

**Comment by arouinfar** [comment]
@samreid good catch! I think this was an oversight rather than a design decision. Unchecking "Stopwatch" should behave the same as returning the stopwatch to the toolbox in Wave Interference. That is, the stopwatch should pause and then reset.

Assigning to @jonathanolson to address the next time this sim is worked on.
